tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,39 +6,51 @@
    "source": [
```

Comment:
<p>You think it's better to number the steps? I don't have a very strong opinion about this but we have to be consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1354/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>